csharp/autobuilder/Semmle.Autobuild/AutobuildOptions.cs

```diff
@@ -107,6 +107,14 @@ public static string[] AsListWithExpandedEnvVars(this string value, IBuildAction
 
         static readonly Regex linuxEnvRegEx = new Regex(@"\$([a-zA-Z_][a-zA-Z_0-9]*)", RegexOptions.Compiled);
 
+        public static string? AsStringWithExpandedEnvVarsMaybeNull(this string? value, IBuildActions actions)
```

Comment:
Couldn't we remove this method and replace uses with `?.AsStringWithExpandedEnvVars(actions)`?

Reply:
Yes that's better.